### ISSUE TITLE
NVDA does not speaks in settings screen of windows 8/8.1/10

### ISSUE THREAD

**Issue by abdullah5490** [nvaccess/nvda#5494]
there is no way to read the screen at modern settings windows in above mentioned OS's. because screen review function doesn't work there. it make certain information unaccessible from NVDA user. for example, while updating windows10, it is not possible to know the downloading status or percentage within the settings windows with NVDA. same problem occurs in modern application that are also called windows store apps.

**Comment by josephsl** [comment]
Hi,

 

Although screen review doesn’t work, object navigation still works, and one may need to use object nav commands to use these kind of apps (this affects not just NVDA but all screen readers). Also, the styles of these apps and placement of description text is such that it is hard for screen readers to read just the needed information, and the only way to force NVDA to read everything is using speak foreground window command (NVDA+B).

 

Thanks.

**Comment by abdullah5490** [comment]
NVDA also don’t read the windows completely by pressing NVDA+b as well there.

**Comment by josephsl** [comment]
Hi,

 

Can you tell me which parts of Windows 8, 8.1 or 10 cannot be read by pressing NvDA+B (as in which part of PC Settings/Settings app, any app that has this problem and so on)? From my own testing, NVDA+B works in most apps of this kind.

Cheers,

Joseph

**Comment by abdullah5490** [comment]
Ok, go to the windows update in windows10 from the settings, here, how you can check the status of update whether it is downloading, or checking for updates, or the message of no update available, last check today at xxx time? It doesn’t read by NVDA with the above command.

**Comment by josephsl** [comment]
Hi,

 

For those, you need to use object navigation, as NVDA does not detect that this text has happened (I’m not sure if UIA is somehow involved in this, as they are UIA controls).

 

By the way, is this request a duplicate of another issue?

**Comment by abdullah5490** [comment]
No, this is not a duplicate. And also, can u please check it at your end by object navigation? And then explain the keystrokes which can access the required info from screen? Because i am unable to figure that out according to your comment. Thanks.

**Comment by josephsl** [comment]
Hi,

 

The commands are as follows:

*         Previous control/object: NVDA+Numpad 4 (left arrow)

*         Next object: NVDA+Numpad 6 (right arrow)

*         Parent control: NVDA_Numpad 8 (up arrow)

*         Child control: NVDA+Numpad 2 (down arrow)

*         Current control/object: NvDA+Numpad 5 (center)

 

Hope this helps.

**Comment by jcsteh** [comment]
You can definitely use object navigation to get to all of this information. See [the object navigation section of the User Guide](http://www.nvaccess.org/files/nvda/documentation/userGuide.html#toc33) for details. It's not possible to make screen review work in modern apps.
